Resources/Prototypes/_SpaceCats/Entities/Clothing/Hands/gloves.yml

```
@@ -0,0 +1,21 @@
+    sprite: _SpaceCats/Clothing/Hands/white_captain_gloves.rsi
+
+- type: entity
+  parent: ClothingHandsGlovesCaptain
+  id: ClothingHandsGlovesSheriff
+  name: sheriff gloves
+  description: Gloves with an ergonomic shape designed to hold a revolver.
+  components:
+  - type: Sprite
+    sprite: _SpaceCats/Clothing/Hands/sheriff_gloves.rsi
+  - type: Clothing
+    sprite: _SpaceCats/Clothing/Hands/sheriff_gloves.rsi
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)